src/player.h

```diff
@@ -1461,6 +1461,12 @@ class player : public Character, public JsonSerializer, public JsonDeserializer
         std::set<tripoint> get_path_avoid() const override;
 
         /**
+         * Makes the NPC considered to be attacked by the player first if not already hostile.
+         * Angers the NPC to allow for fight or flight.
+         */
+        void aggress_npc( npc &target );
```

Comment:
I had gone back and forth on this and I do agree. Based on functionality, it only examines NPC state. However, the action is triggered from the player point of view. I'll move this any try to rename accordingly.